docs/one-click-integrations/jira-integration.md

```diff
@@ -1,29 +1,96 @@
 # Jira integration
 
-Pulse integrates with Jira Cloud to receive data about issues, necessary to calculate the metrics [Lead time and Cycle time](../metrics/lead-cycle-time.md).
+Pulse integrates with Jira Cloud to receive data about:
+
+-   **Issues** - necessary to calculate [Lead time and Cycle time](../metrics/lead-cycle-time.md).
+
+-   **Incidents** - necessary to calculate the following Accelerate metrics:
+
+    -   [Time to recover](../metrics/accelerate.md#time-to-recover)
+
+    -   [Change failure rate](../metrics/accelerate.md#change-failure-rate)
 
 ## Setting up the Jira integration
 
+!!! note
+    -   The Jira integration must be set up by a **Jira administrator**, as Pulse will only have access to Jira resources that the user setting up the integration has access to.
```

Comment:
style: Edit to use active voice. If there are some other similar sentences in the docs, it's better to just keep the consistency for now and do these improvements later in a separate pull request.

```suggestion
    -   A **Jira administrator** must set up the integration, as Pulse will only have access to Jira resources that the user setting up the integration has access to.
```